surveys/2025/annual-survey/translations/de.md

```diff
@@ -300,6 +326,10 @@ Type: select one
 
 ### Welche nicht implementierten oder (nur nächtlichen) Funktionen möchten Sie stabilisieren?
 
+Bitte erwähnen Sie hier Funktionen für den *Rust-Compiler*, die sich derzeit hinter einem Feature-Gate befinden (nur verfügbar mit einer instabilen/nächtlichen Veröffentlichung des Rust-Compilers) oder eine Funktion, die fehlt und Ihrer Meinung nach für das Rust-Ökosystem von Vorteil wäre Ihre Arbeit. Diese Liste schließt andere Tools rund um den Compiler wie Cargo, Rustup, Rustfmt usw. aus.
```

Comment:
For consistency should probably use plural "Funktionen" in the second part as well, and have to fix a grammar issue:
```suggestion
Bitte erwähnen Sie hier Funktionen für den *Rust-Compiler*, die sich derzeit hinter einem Feature-Gate befinden (nur verfügbar mit einer instabilen/nächtlichen Veröffentlichung des Rust-Compilers) oder ein Funktionen, die fehlen und Ihrer Meinung nach für das Rust-Ökosystem von Vorteil für Ihre Arbeit wären. Diese Liste schließt andere Tools rund um den Compiler wie Cargo, Rustup, Rustfmt usw. aus.
```